src/stratus-application/src/main/java/stratus/StratusInitializer.java

```diff
@@ -29,26 +33,52 @@
     @Autowired
 	private ConfigurableApplicationContext app;
 
-    @Autowired(required=false)
-	private Collection<Initializer> initializers;
+    @Autowired
+	private GeoServer geoServer;
```

Comment:
Please don't mix tabs and spaces

Reply:
I thought I had just copied and pasted over these changes from what you made in BSE 1.4.1, but I could be wrong.  Anyway, wasn't intentional.

Reply:
It could very well have been some rogue formatting in 1.4.1, we don't have any programmatic checks or anything.